### PR TITLE
[WP#50326]Added trailing slash for folder for files info endpoint

### PR DESCRIPTION
**Pull request by SagarGi**
### Description
This PR adds the trailing slash for the folder path in the response for the file(s)info endopint.

### Relared WP
https://community.openproject.org/projects/nextcloud-integration/work_packages/50326/activity

### CODE REVIEW

File: tests/acceptance/features/api/getFileinfoByFileIDAPI.feature [hunk 1]
```diff
@@ -1260,7 +1260,7 @@ Feature: retrieve file information of a single file, using the file ID
       "modifier_id": {"type": "null"},
       "modifier_name": {"type": "null"},
       "dav_permissions": {"type": "string", "pattern":"^RMGDNVCK$"},
-      "path": {"type": "string", "pattern":"^files/groupFolder$"}
+      "path": {"type": "string", "pattern":"^files/groupFolder/$"}
```
Comment (SwikritiT) [inline tests/acceptance/features/api/getFileinfoByFileIDAPI.feature:1263]:
```suggestion
      "path": {"type": "string", "pattern":"^files\/groupFolder\/$"}
```

File: tests/acceptance/features/api/getFileinfoByFileIDAPI.feature [hunk 2]
```diff
@@ -900,7 +900,7 @@ Feature: retrieve file information of a single file, using the file ID
       "modifier_name": {"type": "null"},
       "trashed": {"type": "boolean", "enum": [false]},
       "dav_permissions": {"type": "string", "pattern":"^RGDNVCK$"},
-      "path": {"type": "string", "pattern":"^files\/folder$"}
+      "path": {"type": "string", "pattern":"^files\/folder/$"}
```
Comment (SwikritiT) [inline tests/acceptance/features/api/getFileinfoByFileIDAPI.feature:903]:
```suggestion
      "path": {"type": "string", "pattern":"^files\/folder\/$"}
```

File: tests/acceptance/features/api/getFileinfoByFileIDAPI.feature [hunk 3]
```diff
@@ -1173,7 +1173,7 @@ Feature: retrieve file information of a single file, using the file ID
       "modifier_id": {"type": "null"},
       "modifier_name": {"type": "null"},
       "dav_permissions": {"type": "string", "pattern":"^<requester-dav-permissions>$"},
-      "path": {"type": "string", "pattern":"^files\/folder$"}
+      "path": {"type": "string", "pattern":"^files\/folder/$"}
```
Comment (SwikritiT) [inline tests/acceptance/features/api/getFileinfoByFileIDAPI.feature:1176]:
```suggestion
      "path": {"type": "string", "pattern":"^files\/folder\/$"}
```

File: tests/acceptance/features/api/getFileinfoByFileIDAPI.feature [hunk 4]
```diff
@@ -1205,7 +1205,7 @@ Feature: retrieve file information of a single file, using the file ID
       "modifier_id": {"type": "null"},
       "modifier_name": {"type": "null"},
       "dav_permissions": {"type": "string", "pattern":"^<owner-dav-permissions>$"},
-      "path": {"type": "string", "pattern":"^files\/folder$"}
+      "path": {"type": "string", "pattern":"^files\/folder/$"}
```
Comment (SwikritiT) [inline tests/acceptance/features/api/getFileinfoByFileIDAPI.feature:1208]:
```suggestion
      "path": {"type": "string", "pattern":"^files\/folder\/$"}
```

File: tests/lib/Controller/FilesControllerTest.php [hunk 5]
```diff
@@ -79,7 +83,8 @@ public function getFileInfoDataProvider() {
 				],
 				'files',
 				'files',
-				'application/x-op-directory'
+				'application/x-op-directory',
+				'files/test/'
```
Comment (individual-it) [inline tests/lib/Controller/FilesControllerTest.php:87]: the expectation here should be `files/`

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by individual-it — APPROVED** [review]

**Review by SwikritiT — APPROVED** [review]

**Pull request merged**